samples/zephyr-hello/README.md

```diff
@@ -0,0 +1,29 @@
+# Zephyr Hello World
+
+Hello World sample for embedded devices based on [Zephyr](https://zephyrproject.org/)
+
+## Installation
+
+1. Checkout the source code of Zephyr:
+
+        git clone https://github.com/zephyrproject-rtos/zephyr.git
+
+2. Install Zephyr dependencies as described in
+http://docs.zephyrproject.org/getting_started/getting_started.html
+
+3. If you can succesfully run
+http://docs.zephyrproject.org/getting_started/getting_started.html#running-a-sample-application-in-qemu
```

Comment:
https://docs.zephyrproject.org/getting_started/getting_started.html#run-the-application-in-qemu